### PR TITLE
Issue 5 user can click to search skins

### PR DESCRIPTION
**Pull request by restropiajr**
Recording 2023-06-16 at 01 34 38
Recording 2023-06-16 at 01 35 34

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -138,6 +138,20 @@ <h4>Conquer the Terrain and Control the Game</h4>
       </div>
     </div>
 
+    <div class="container" data-view="skins-page">
+      <div class="row">
+        <div class="skin-title title column-full">
+          <h1>SKINS</h1>
+          <h4>Elevate Your Look and Explore the Vibrant World of Skins in Valorant</h4>
+          <form class="search-form">
+            <input id="search-bar" placeholder="Search a weapon category or skin"></label>
```
Comment (shawnkost) [inline index.html]: You have a closing `label` tag, but there's no opening `label` tag. Remove the `</label>`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Roland!

File: index.html (hunk 1 above)
Comment (restropiajr) [reply index.html]: I thought I removed it but it must of undid when I did some redos. Thanks for the catch!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**